react-site/src/components/Name.js

```diff
@@ -0,0 +1,21 @@
+import '../App.css';
+import React, {Component} from 'react';
+
+class Name extends Component{
+    constructor(props){
+        super(props);
+        this.state = {name:""}
+    }
+
```

Comment:
this would actually be a great place to use props! you could pass in the name from the input box in app.js and render it here!

nice job making another component